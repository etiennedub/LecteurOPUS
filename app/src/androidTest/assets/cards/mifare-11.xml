<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">04146FF7</List><List index="1">E2664980</List><List index="2">4D48E000</List><List index="3">80000000</List><List index="4">00000000</List><List index="5">300B4523</List><List index="6">71ABDB30</List><List index="7">BB487E02</List><List index="8">01C6A3FC</List><List index="9">8153C300</List><List index="10">650038D4</List><List index="11">00281885</List><List index="12">01C6A3FC</List><List index="13">56A6050B</List><List index="14">2E0038D4</List><List index="15">002833B7</List></Pages></Mifare>