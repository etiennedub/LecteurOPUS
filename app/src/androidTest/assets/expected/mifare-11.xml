<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="01/12/2016" Time="17:14" Transfert="false" OperatorId="20" BusId="316" OperatorName="RTC" BusName="" /><List index="1" Date="01/12/2016" Time="11:33" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /></Trip><Contract><List index="0" Subscription="true" Valid="false" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>
